app-editors/emacs/emacs-30.1-r1.ebuild

```diff
@@ -32,15 +32,22 @@ else
 		SRC_URI="https://alpha.gnu.org/gnu/emacs/pretest/${PN}-${PV/_/-}.tar.xz"
 	fi
 	SLOT="${PV%%.*}"
-	[[ ${PV} == *.*.* ]] && SLOT+="-vcs"
-	KEYWORDS="~alpha amd64 arm ~arm64 hppa ~loong ~m68k ~mips ppc ppc64 ~riscv sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos"
+	if [[ ${PV} == *.*.* ]]; then
+		SLOT+="-vcs"
+	else # normal release
+		IUSE="verify-sig"
+		SRC_URI+=" verify-sig? ( mirror://gnu/emacs/${P}.tar.xz.sig )"
+		VERIFY_SIG_OPENPGP_KEY_PATH=/usr/share/openpgp-keys/${PN}.asc
+	fi
```

Comment:
1. I don't think this conditional is necessary, because upstream prerelease versions are signed too. So having the exceptions for the live version (and maybe `_pre` snapshots) should be enough.
2. The eclass already assigns `IUSE="verify-sig"`, so it is not needed here.